cmd/kube-apiserver/app/aggregator.go

```diff
@@ -349,3 +317,10 @@ func apiServicesToRegister(delegateAPIServer genericapiserver.DelegationTarget,
 
 	return apiServices
 }
+
+func merge(a, b map[schema.GroupVersion]controlplaneapiserver.APIServicePriority) map[schema.GroupVersion]controlplaneapiserver.APIServicePriority {
```

Comment:
now it's not a problem, but I was first thinking that it could be problematic if this function would be passed some generic global variable, as this func mutates `a`. You indeed solved it with having a function returning a new variable every time, but just something to be careful about, unless this function first does a deep copy of a.